client/lib/Screens/login/login_screen.dart

```diff
@@ -0,0 +1,187 @@
+// ignore_for_file: prefer_const_constructors
+
+import 'package:flutter/material.dart';
+
+class LoginScreen extends StatelessWidget {
+  const LoginScreen({Key? key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      backgroundColor: Color(0xffFFFFFF),
+      body: ListView(
+        padding: EdgeInsets.symmetric(
+            horizontal: MediaQuery.of(context).size.width / 8),
+        children: const [
+          Menu(),
+          Body(),
+        ],
+      ),
+    );
+  }
+}
+
+class Menu extends StatelessWidget {
+  const Menu({Key? key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return Padding(
+      padding: const EdgeInsets.symmetric(vertical: 30),
```

Comment:
To have a consistency between all screens, could you store all dimensions in a separated file please?

Reply:
Why do you mark this as resolved, as the dimension is still hardcoded?

Reply:
how I can put in a centralized file? Is creating a const for appappding would be good in theme_constants file?